src/components/view/index.tsx

```diff
@@ -32,7 +32,7 @@ export interface ViewPropTypes extends Omit<ViewProps, 'style'>, ContainerModifi
    * Set background color
    */
   backgroundColor?: string;
-  style?: StyleProp<ViewStyle> | Animated.AnimatedProps<StyleProp<ViewStyle>>;
+  style?: StyleProp<ViewStyle | Animated.AnimatedProps<ViewStyle>>;
```

Comment:
I almost sure I changed that because of an issue I had in the private repo. 
Is there a way to verify it doesn't break anything in the private repo?

Reply:
Do you mean break the TS or usage?

Reply:
break TS, I don't think it can break functionality..

Reply:
I pasted the change into `view/index.d.ts` and it didn't break `npm run build:dev`, not sure if that's enough of a check

Reply:
That's should be enough. merging.